### ISSUE TITLE
Coke Oven Render problem

### ISSUE THREAD

**Issue by betacyth** [BluSunrize/ImmersiveEngineering#476]
Not sure how to explain it so I am including three images of the Coke Oven
![2015-09-30_13 45 50](https://cloud.githubusercontent.com/assets/14912619/10203106/cb15ce1a-6779-11e5-82dd-e317988ab97e.png)
![2015-09-30_13 45 54](https://cloud.githubusercontent.com/assets/14912619/10203108/cb19b55c-6779-11e5-9a8c-54300a520c3e.png)
![2015-09-30_13 45 57](https://cloud.githubusercontent.com/assets/14912619/10203107/cb198a00-6779-11e5-9551-dc2a66239789.png)

**Comment by cobra** [comment]
What version is this? Are you using optifine or fastcraft?
Also: see #474.

**Comment by betacyth** [comment]
0.5.4 IE
And Optifine 1.7.10 HD U C1

**Comment by liamjarvisman** [comment]
optifine might be the problem i don't think IE suports it.

**Comment by BluSunrize** [comment]
And IE will never support it. Optifine breaks all the things.

**Comment by liamjarvisman** [comment]
to true :)

**Comment by betacyth** [comment]
But if it's optifine then why is the other player having the same problems as me while running fastcraft? I was able to replicate everything they reported on.

**Comment by BluSunrize** [comment]
@betacyth you did not tell us that you could replicate this without optifine.

The lack of particles is defitnitely my bad, but the weirdly placed block icons I have yet to see reported without Optifine.

**Comment by GirafiStudios** [comment]
I have the same issues when looking at the coke oven from certain angles - and i'm not using Optifine. I'm using Fastcraft tho.

**Comment by XFactHD** [comment]
Possibly another mod is messing with texture sizes and/or metadata based texture positioning.

**Comment by liamjarvisman** [comment]
I don't think its fast craft because i run fast craft and the coke oven is fine

**Comment by liamjarvisman** [comment]
try and remember if you added a new mod lately so then we could see if that mod is causing the problem when i install it.

**Comment by betacyth** [comment]
Problem with that is IE is the mod I added to my pack. Been dying to use it since I saw on Forge craft 2. Any ideas on how I could find a conflict efficiently? (Have over 130 mods.) I myself haven't tried it without optifine (just saw someone with similar/same problem. I was able to replicate what they reported and they were using fast craft. I'll try it without optifine when I get back from work in a few hours. In the meantime does anyone know of any current mod conflicts between IE and another mod (perhaps common mods commonly with problems?)
